packages/direflow-component/src/WebComponentFactory.tsx

```diff
@@ -207,8 +208,9 @@ class WebComponentFactory {
           currentChildren = Array.from(this.children).map((child: Node) => child.cloneNode(true));
         }
 
-        const root = createProxyRoot(this);
-        ReactDOM.render(<root.open>{application}</root.open>, this);
+        const wrappedAppAndMountPoint = handleMaterialUiStyle(application, factory.plugins);
```

Comment:
I'm not able to build due to typing related problems here.
`Argument of type 'Element' is not assignable to parameter of type 'FC<any> | ComponentClass<any, any>'.
  Type 'Element' is not assignable to type 'ComponentClass<any, any>'.`